components/ui/Icon/index.js

```diff
@@ -207,7 +208,8 @@ const mapNameIcons = {
     review: reviewIcon,
     reviewHalf: reviewHalf,
     reviewSmall: reviewSmall,
-    reviewHalfSmall: reviewHalfSmall
+    reviewHalfSmall: reviewHalfSmall,
+    goBack: goBack
```

Comment:
This icon already exists here and should not be created again

https://www.[unzipped.io/subscribe](https://www.unzipped.io/subscribe)